### ISSUE TITLE
Migrate to GeoBlacklight Sidecar Images 0.6.1

### ISSUE THREAD

**Issue by ewlarson** [BTAA-Geospatial-Data-Project/geoportal#153]
After #136 migrate to ActiveStorage based imagery by updating to the GBLSCI 0.50 release

**Comment by ewlarson** [comment]
Perhaps helpful: https://stackoverflow.com/questions/52528623/migrating-carrierwave-to-activestorage

**Comment by ewlarson** [comment]
Notes on migration workflow

### Following pattern here:
https://stackoverflow.com/questions/52528623/migrating-carrierwave-to-activestorage

```bash
bundle exec rails active_storage:install
bundle exec rails db:migrate
```

```ruby
# Add to solr_document_sidecar.rb
has_one_attached :image_new
```

### Confirm current state
```base
bundle exec rake geoportal:queue_incomplete_states
bundle exec rake geoportal:sidecar_states
```

### Local results

* initialized - 1 (orphaned record)
* queued - 0
* processing - 0
* succeeded - 25
* failed - 3
* placeheld - 17

```ruby
# Test migrate task
# Attaches Carrierwave image to ActiveStorage
# @TODO - will double file space used
SolrDocumentSidecar.find_each do |sidecar|
  begin
    sidecar.image_new.attach(io: open(sidecar.image.file.file), filename: sidecar.image.file.filename)
  rescue => e
    puts e.inspect
  end
end

# @TODO - checks to see if image was migrated
SolrDocumentSidecar.find_each do |sidecar|
  puts "#{sidecar.id} - #{sidecar.state_machine.current_state} - #{sidecar.image_new.attached?}"
  begin
    puts Rails.application.routes.url_helpers.rails_blob_path(sidecar.image_new, only_path: true)
  rescue => e
    puts e.inspect
  end
end
```

**Comment by ewlarson** [comment]
Closing. Released with 2.2.6 and 2.2.7.
